postgreSchema.js

```diff
@@ -27,70 +27,70 @@ await client.queryArray(
   `INSERT INTO scores
   (score, created_at, word, username)
    VALUES
-   (3, CURRENT_DATE, 'choke', 'player1')
+   (3, CURRENT_DATE, 'rupee', 'mixup692')
 ;`
 );
```

Comment:
is this test going to pass if we're on a different day?

Reply:
Those are just seed data, so we can change it for each day or just leave it in and add more!